common/flatpak-dir.c

```diff
@@ -7602,23 +7635,33 @@ flatpak_dir_install (FlatpakDir          *self,
           if (child_repo == NULL)
             return FALSE;
 
-          flatpak_flags |= FLATPAK_PULL_FLAGS_SIDELOAD_EXTRA_DATA;
+          bindfs_repo_file = g_file_new_for_path (bindfs_repo_path);
+          bindfs_repo = flatpak_dir_create_child_repo (self, bindfs_repo_file, &bindfs_repo_lock, NULL, error);
+          if (bindfs_repo == NULL)
+            {
+              g_warning ("Cannot create child repo on given path");
```

Comment:
```c
g_prefix_error (error, _("Cannot create child repo at %s: "), bindfs_repo_path);
```

might be better?

Reply:
hmm, As per my changes in  b55725c , I still think to display a warning here. The idea is, if we fail on bindfs code-path, we just don't want to error out; instead fallback on existing code-path and display a warning (that there was something wrong with bindfs code path). Does it make sense?

Reply:
I think the code change there makes sense, modulo my comment on that commit.